lib/mina-circle/helpers.rb

```diff
@@ -2,9 +2,15 @@ module MinaCircle
   module Helpers
     def build_url
       latest_build = circle_ci.builds.first
-      circle_ci.artifacts(latest_build).select { |artifact|
+      artifacts = circle_ci.artifacts(latest_build).select { |artifact|
         artifact.filename == circle_artifact
-      }.first.url
+      }
+      raise "No artifacts found for build: #{latest_build.build_num} on #{branch}. #{latest_build.status}" if artifacts.empty?
+      artifacts.first.url
+    rescue Exception => e
+      puts e.message
+      puts ""
+      puts "Review your build at to ensure artifacts are being captured as described in https://circleci.com/docs/build-artifacts/"
```

Comment:
Are we really throwing an exception just to catch it? Using exceptions for flow control?